coherence/upnp/core/service.py

```diff
@@ -370,6 +370,13 @@ def gotPage(x):
                     {urn:schemas-beebits-net:service-1-0}X_withVendorDefines
                     attibute there
                 """
+                allowed_range = var_node.find('{%s}allowedValueRange' % ns)
+                if allowed_range:
+                    allowed_range_dict = {}
+                    for item in allowed_range:
+                        namespace_uri, tag = item.tag[1:].split("}", 1)
```

Comment:
Shouldn't we verify what we are reading here? Otherwise some device could send some "}garbage" tag.